README.md

```diff
@@ -142,9 +143,17 @@ print("Ran in {} seconds".format(time.time() - start))
 
 ```
 
+## Building PlaidML
+
+PlaidML depends on [Bazel](http://bazel.build) v0.6.0 and higher.
+```
+bazel build -c opt plaidml:wheel plaidml/keras:wheel
+sudo pip install -U bazel-bin/plaidml/*whl bazel-bin/plaidml/keras/*whl
+```
+
 ## License
 
-PlaidML is licensed under the [AGPLv3](https://www.gnu.org/licenses/agpl-3.0.txt).
+PlaidML is licensed under the [AGPLv3](https://www.gnu.org/licenses/agpl-3.0.txt). Commercial licenses and support for PlaidML are available from [Vertex.AI](mailto:info@vertex.ai).
```

Comment:
"For alternative licensing contact [solutions@vertex.ai](mailto:solutions@vertex.ai)